docs/src/pages/components/buttons/LoadingButtons.tsx

```diff
@@ -1,21 +1,17 @@
 import * as React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
 import LoadingButton from '@material-ui/lab/LoadingButton';
 import SaveIcon from '@material-ui/icons/Save';
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    '& button': {
-      margin: theme.spacing(1),
-    },
-  },
-}));
-
 export default function LoadingButtons() {
-  const classes = useStyles();
-
   return (
-    <div className={classes.root}>
+    <Box
+      sx={{
+        '& > button': {
```

Comment:
```suggestion
        '& > :not(style)': {
```

Reply:
Done. Corrections are made to both files (.js and .tsx)